build.gradle.kts

```diff
@@ -67,6 +79,16 @@ kotlin {
             }
         }
     }
+    val linuxX64 by sourceSets.creating {
+        dependsOn(nativeMain)
+    }
+    targets.withType(KotlinNativeTarget::class.java) {
```

Comment:
This can just be `targets.withType<KotlinNativeTarget> {`.

Reply:
Switched